<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QComboBox" name="preferenceBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
  </widget>
  <widget class="QScrollBar" name="scrollPeople">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
   <property name="sliderPosition">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEmailCopy">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy Email List to clipboard</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Full Name</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Postal Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Email Address</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="postBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="numP">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="prayerLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Prayer Letters</string>
   </property>
  </widget>
  <widget class="QLabel" name="termLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Term Cards</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numT">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="financeLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Finance Letters</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numF">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
